### DISCUSSION TITLE
Required status pass but blocks from merging Pull Request

### DISCUSSION THREAD

**Discussion by cscleison** [community/community#25876] [Actions]
<p>Yesterday we migrated our workflows from HCL to yml syntax, all worked well.</p>
<p>Since this morning, our required workflow finishes correctly but the UI won’t let me merge the PR.</p>
<p>The status that pass is <strong>required</strong> (the UI is not showing the required label anymore) and it’s a Github Action.</p>
<p>The failed status is not an github action and is not required. </p>
<p>I should be able to merge the PR but since this morning we can’t.</p>
<p>If I remove the requirement for the green workflow, then I’m able to merge.</p>
<p><img src="https://user-images.githubusercontent.com/4950497/181093797-423c7ddc-bcdc-43f3-a6a4-964be96fed97.png" alt="Annotation 2019-10-02 135703.png" data-base62-sha1="ym3sTFkAmUow2avawbSgY68eNoy" width="689" height="470"></p>

**Comment by SamChou19815** [comment]
<p>I’m experiencing the same issue. Example: <a href="https://github.com/cornell-dti/trends-in-web-dev-website/pull/20" rel="nofollow noopener">https://github.com/cornell-dti/trends-in-web-dev-website/pull/20</a></p>

**Comment by chrispat** [comment, marked as answer]
<p>There was an issue we found that should now be fixed.  Please let me know if you are still seeing this problem.</p>
